components/account/login/LoginModal.vue

```diff
@@ -1,26 +1,23 @@
 <script setup lang="ts">
-defineProps<{ modelValue: boolean }>();
+const props = defineProps<{ modelValue: boolean }>();
 
 const emit = defineEmits<{
   (e: 'update:modelValue', dismiss: boolean): void;
 }>();
 
-function dismiss() {
-  emit('update:modelValue', false);
-}
+const vModel = useSimpleVModel(props, emit);
```

Comment:
we are already in vue 3.4 here right? so we should be able to do 
```typescript
const modelValue = defineModel<boolean>({required: true})
```

Reply:
updated, what  a world to live in